src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Lazy-loading is a prickly pear
+subhead: Optimizing for performance without the sharp edges.
+description: Recent research has shown that lazy loading images above-the-fold can lead to regressions in Core Web Vitals performance.
+authors:
+  - rviscomi
+  - felixarntz
+date: 2021-07-14
+hero: image/STd8eW8CSiNp5B1bX0R6Dww2eH32/h5OlDz9eOONtAi8JNqh7.jpg
```

Comment:
this image doesn't look great in the preview.

Reply:
Reuploaded a higher res image

Also ironically web.dev lazy loads the hero image!! 😆